grr/server/grr_response_server/gui/ui/lib/api/http_api_service.ts

```diff
@@ -223,6 +223,14 @@ export class HttpApiService {
         url, {client_ids: [clientId], labels: [label]});
   }
 
+  removeClientLabel(clientId: string, label: string): Observable<HttpResponse<{}>> {
```

Comment:
HttpApiResponse is an implementation detail and should not be returned. Instead, return {} or void.